lib/rule-context.js

```diff
@@ -81,6 +82,7 @@ function RuleContext(ruleId, eslint, severity, options, settings, parserOptions,
     this.parserOptions = parserOptions;
     this.parserPath = parserPath;
     this.meta = meta;
+    this.parserServices = parserServices;
```

Comment:
Just wondering, should this be frozen with `Object.freeze()` to avoid rules making modifications?